src/Imports/ModelManager.php

```diff
@@ -212,4 +202,13 @@ private function rows(): Collection
     {
         return new Collection($this->rows);
     }
+
+    private function handleException(ToModel $import, Throwable $e): void
+    {
+        if (! $import instanceof SkipsOnError) {
```

Comment:
There is only one small phpcs fail here
```suggestion
        if (!$import instanceof SkipsOnError) {
```